stdlib/system/src/main/java/org/ballerinalang/stdlib/system/nativeimpl/CreateFile.java

```diff
@@ -58,12 +59,18 @@ public static Object createFile(Strand strand, String path) {
             String msg = "Permission denied. Failed to create the file: " + path;
             log.error(msg, e);
             return SystemUtils.getBallerinaError(SystemConstants.PERMISSION_ERROR, msg);
+        } catch (NoSuchFileException e) {
+            String msg = "The file does not exist in path " + path;
+            log.error(msg, e);
```

Comment:
We don't need to log this. This is a straight forward issue.

Reply:
Resolved